corehq/apps/app_manager/templates/app_manager/partials/define_case_type_modal.html

```diff
@@ -0,0 +1,80 @@
+{% load i18n %}
+{% load hq_shared_tags %}
+
+<div
+  class="modal fade define-case-type-modal"
+  tabindex="-1"
+  role="dialog"
+  id="define-case-type-modal"
+>
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header" style="border-bottom: 1px solid lightgray;">
```

Comment:
Inline styles are always a bit concerning. Do we want to think about better ways to handle this, or perhaps this is unnecessary?

Reply:
I agree with this but also don't really care since it'll get flagged in the bootstrap migration.

Reply:
Talked with Biyeun, since we already define border for modal-header in new module modal, it would be weird if the case type modal suddenly don't have border. I left a comment in 13ae7b64894a3d6d0a6ca66f40a98d08adaeb6a5 for this inline style to address with b5 utility when we migrated to b5.